### DISCUSSION TITLE
How to skip a region?

### DISCUSSION THREAD

**Discussion by XyzAQWERTY** [katspaugh/wavesurfer.js#3091] [Q&A] [answered]
Can we enable region skip? So the media will skip on the region selection
![image](https://github.com/katspaugh/wavesurfer.js/assets/129366472/c6b14838-5638-498d-8581-0cfa33255ab1)

**Comment by katspaugh** [comment, marked as answer]
You can subscribe to the region-in event and call `wavesurfer.setTime(region.end)`.
